### PR TITLE
New Detection - KrbRelayUp Tool

### PR DESCRIPTION
**Pull request by BlackB0lt**
Required items, please complete

Change(s):

Create New rule to detect service creation from KrbRelayUp tool
Reason for Change(s):

New rule to detect service creation from KrbRelayUp tool
Version Updated: 1.0

Required only for Detections/Analytic Rule templates
Testing Completed:

Tested on event with sigma rule
Checked that the validations are passing and have addressed any issues that are present:

See guidance below
References: https://github.com/Dec0ne/KrbRelayUp

### CODE REVIEW

File: Hunting Queries/SecurityEvent/KrbRelayUpServiceCreation [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1543
+query: |
+  let MaliciousService = dynamic (["KrbSCM"]);
+  Event
+  | where Source == "Service Control Manager" and EventID == 7045
+  | parse EventData with * 'ServiceName">' ServiceName "<" * 'ImagePath">' ImagePath "<" *
+  | where ServiceName has_any (MaliciousServiceArtifacts) or ImagePath has_any (MaliciousServiceArtifacts)
```
Comment (aprakash13) [inline Hunting Queries/SecurityEvent/KrbRelayUpServiceCreation:23]: Seems like we should have MaliciousService vs  MaliciousServiceArtifacts in here.

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Please take a quick look at the suggested changes.

**Comment by BlackB0lt** [comment]
This detection is related with this article.

https://www.microsoft.com/security/blog/2022/05/25/detecting-and-preventing-privilege-escalation-attacks-leveraging-kerberos-relaying-krbrelayup/

**Review by aprakash13 — APPROVED** [review]
Thanks @BlackB0lt

**Pull request merged**